### ISSUE TITLE
Export an existing content model from Drupal

### ISSUE THREAD

**Issue by darkcody** [acquia/drupal-spec-tool#27]
Hey @TravisCarden , I'd like to be able to export an existing spec. My use case: I've inherited an existing codebase and would like to use the drupal spec tool to document future content model changes and generate behat tests, but I don't want to spend the time to manually enter the necessary information into the sheet.

I recently took a stab at such a feature. I'm happy to clean it up and submit a pull request if its something you'd consider. It adds two step definitions to the ContentModelContext, invoked in the behat test scenario as "When I export content entity type bundles" and "When I export content entity type fields". These each export a csv that can be imported into google sheets.

**Comment by TravisCarden** [comment]
Hello, @darkcody. Nice to see you in my queue! 😃 I'm _definitely_ interested in this functionality but as some sort of CLI command rather than a Behat step. @NickWilde1990 has created something very similar in his [drupal-spec-tool-commands](https://github.com/NickWilde1990/drupal-spec-tool-commands), which update your Behat feature files to the spreadsheet, so there's probably an opportunity here for code reuse if not collaboration. In fact, I've had some time to think about how I want to handle CLI commands since he proposed his (see https://github.com/acquia/drupal-spec-tool/pull/24). Why don't I file an issue to create the CLI framework, and then we can look at adding specific features like this one?

**Comment by NickWilde1990** [comment]
Definitely, very busy this week, but I'll keep my ears open and am available to talk anytime
